system-tests/tia-maven/maven-dump-local-project/pom.xml

```diff
@@ -47,13 +47,9 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <teamscaleUrl>http://localhost:${tia.teamscale.fake.port}</teamscaleUrl>
                     <projectId>m</projectId>
-                    <username>build</username>
-                    <accessToken>6lJKEvNHeTxGPhMAi4D84DWqzoSFL1p4</accessToken>
-                    <endCommit>master:HEAD</endCommit>
                     <partition>MyPartition</partition>
```

Comment:
How does the plugin behave if you don't give it a partition nor a projectId? Can it deal with that or does it at least produce a proper error message?

Reply:
The project ID was mandatory, so far. I have changed that. For the Partition, there are default values (always have been).